pkg/appfile/storage/driver/local.go

```diff
@@ -0,0 +1,131 @@
+package driver
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/ghodss/yaml"
+
+	"github.com/oam-dev/kubevela/pkg/appfile"
+	"github.com/oam-dev/kubevela/pkg/appfile/template"
+	"github.com/oam-dev/kubevela/pkg/utils/env"
+	"github.com/oam-dev/kubevela/pkg/utils/system"
+)
+
+// LocalDriverName is local storage driver name
+const LocalDriverName = "Local"
+
+// Local Storage
+type Local struct {
+}
+
+// NewLocalStorage get storage client of Local type
+func NewLocalStorage() *Local {
+	return &Local{}
+}
+
+// Name  is local storage driver name
+func (l *Local) Name() string {
+	return LocalDriverName
+}
+
+// List applications from local storage
+func (l *Local) List(envName string) ([]*RespApplication, error) {
+	appDir, err := getApplicationDir(envName)
+	if err != nil {
+		return nil, fmt.Errorf("get app dir from env %s err %w", envName, err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("Getting application directory from env %s failed, error: %w", envName, err)
```
Or a complete error message to help users to understand the issue, like `Application directory could not be retrieved from environment...`, `Getting application directory from environment xxx hit an issue:`

Reply:
Done